.drone.yml

```diff
@@ -0,0 +1,89 @@
+clone:
+  git:
+    image: registry.fpfis.eu/drone-plugins/git:next
+    when:
+      event: [ push, pull_request, tag ]
+  git:
+    image: registry.fpfis.eu/drone-plugins/git
+    when:
+      event: [ deployment ]
+
+workspace:
+  base: /test
+  path: oe_content_extra
+
+services:
+  web:
+    image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION}-ci
+    environment:
+      - DOCUMENT_ROOT=/test/oe_content_extra
+      - COMPOSERVER=--2
+  mysql:
+    image: registry.fpfis.eu/fpfis/sql:percona-5.7
+    command: --innodb-log-file-size=1G --max_allowed_packet=1G --innodb-buffer-pool-size=512M --wait_timeout=3000 --net_write_timeout=3000 --log_error_verbosity=3
+    environment:
+      - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+  sparql:
+    # Temporarily fix the triple store version to avoid "Provisional data" terms.
+    # @todo: Use the latest release once a solution for the "Provisional data" terms is implemented.
+    image: registry.fpfis.eu/openeuropa/triple-store-dev:1.14.0
+    pull: true
+    environment:
+      - SPARQL_UPDATE=true
+      - DBA_PASSWORD=dba
+
+pipeline:
+  composer-install-highest:
+    group: prepare
+    image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION}-ci
+    volumes:
+      - /cache:/cache
+    commands:
+      - composer install --ansi --no-progress
```

Comment:
```suggestion
      - composer self-update --2
      - composer install --ansi --no-progress
```

Reply:
We are dropping these ones, as we did in oe_showcase.